<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WicdKcmWidget</class>
 <widget class="QWidget" name="WicdKcmWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>588</width>
    <height>529</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="KTabWidget" name="ktabwidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>General Settings</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Network Interfaces</string>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Wireless Interface</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="KComboBox" name="wirelessBox"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Wired Interface</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="KComboBox" name="wiredBox"/>
          </item>
          <item row="2" column="0">
           <widget class="QCheckBox" name="showWiredBox">
            <property name="text">
             <string>Always show wired interface</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QCheckBox" name="swithcWiredBox">
            <property name="text">
             <string>Always switch to a wired connection when available</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Global DNS Servers</string>
         </property>
         <layout class="QFormLayout" name="formLayout_2">
          <item row="0" column="0" colspan="2">
           <widget class="QCheckBox" name="useDNSBox">
            <property name="text">
             <string>Use global DNS servers</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>DNS domain</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="KLineEdit" name="domainEdit"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Search domain</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="KLineEdit" name="searchDomainEdit"/>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>DNS Server 1</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="KLineEdit" name="server1Edit"/>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>DNS Server 2</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="KLineEdit" name="server2Edit"/>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>DNS Server 3</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="KLineEdit" name="server3Edit"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>Other</string>
         </property>
         <layout class="QFormLayout" name="formLayout_3">
          <item row="0" column="0">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Wired automatic connection</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="KComboBox" name="automaticWiredBox"/>
          </item>
          <item row="1" column="0">
           <widget class="QCheckBox" name="reconnectBox">
            <property name="text">
             <string>Automatically reconnect on network connection loss</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QCheckBox" name="notificationsBox">
            <property name="text">
             <string>Display notifications about connection status</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Advanced Settings</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QGroupBox" name="groupBox_4">
         <property name="title">
          <string>External Programs</string>
         </property>
         <layout class="QFormLayout" name="formLayout_4">
          <item row="0" column="0">
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>DHCP Client</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="KComboBox" name="dhcpBox"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>Wired Link Detection</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="KComboBox" name="wiredLinkBox"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>Route Table Flushing</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="KComboBox" name="routeBox"/>
          </item>
          <item row="3" column="1">
           <widget class="KComboBox" name="sudoBox"/>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_12">
            <property name="text">
             <string>Graphical Sudo Application</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_5">
         <property name="title">
          <string>Advanced Daemon Settings</string>
         </property>
         <layout class="QFormLayout" name="formLayout_5">
          <item row="0" column="0">
           <widget class="QLabel" name="label_13">
            <property name="text">
             <string>WPA Supplicant Driver</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="KComboBox" name="wpaSupplicantBox"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_14">
            <property name="text">
             <string>Backend</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QCheckBox" name="debugBox">
            <property name="text">
             <string>Enable debug mode</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QCheckBox" name="dbmBox">
            <property name="text">
             <string>Use dBm to measure signal strength</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="KComboBox" name="backendBox"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KTabWidget</class>
   <extends>QTabWidget</extends>
   <header>ktabwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>useDNSBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>domainEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>293</x>
     <y>225</y>
    </hint>
    <hint type="destinationlabel">
     <x>331</x>
     <y>252</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>useDNSBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>searchDomainEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>293</x>
     <y>225</y>
    </hint>
    <hint type="destinationlabel">
     <x>331</x>
     <y>280</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>useDNSBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>server1Edit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>293</x>
     <y>225</y>
    </hint>
    <hint type="destinationlabel">
     <x>331</x>
     <y>308</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>useDNSBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>server2Edit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>293</x>
     <y>225</y>
    </hint>
    <hint type="destinationlabel">
     <x>331</x>
     <y>336</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>useDNSBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>server3Edit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>293</x>
     <y>225</y>
    </hint>
    <hint type="destinationlabel">
     <x>331</x>
     <y>364</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
